.env

```diff
@@ -0,0 +1 @@
+MONGO_URL="mongodb://root:example@0.0.0.0:27017"
```

Comment:
We do not upload the `.env` files to github. Keep the `.env` on your machine, and use viper defaults

Reply:
done